themes/baseTheme/styles/common.json

```diff
@@ -1,2 +1,29 @@
 {
+	".sf": {
```

Comment:
Could you move those classes out of common.json ? Could you do it like it's done on defaultTheme ?